pkg/kubelet/apis/config/types.go

```diff
@@ -514,6 +514,13 @@ type KubeletConfiguration struct {
 	// +featureGate=KubeletCrashLoopBackoffMax
 	// +optional
 	CrashLoopBackOff CrashLoopBackOffConfig
+
+	// SubidsPerPod is the mapping length of subids for UserNS.
+	// The length must be multiple of 65536.
+	// Default: 65536
+	// +featureGate=UserNamespaceSupport
+	// +optional
+	SubidsPerPod uint32
```

Comment:
Should this be in a userns section or some other place, rather than top level config option?

Reply:
Added `userNamespace` structure